feg/protos/BUILD.bazel

```diff
@@ -25,3 +26,61 @@ python_proto_library(
     protos = [":mconfigs_proto"],
     deps = ["//orc8r/protos:common_python_proto"],
 )
+
+proto_library(
+    name = "s6a_proxy_proto",
+    srcs = ["s6a_proxy.proto"],
+)
+
+cpp_proto_library(
+    name = "s6a_proxy_cpp_proto",
+    protos = [":s6a_proxy_proto"],
+)
+
+cpp_grpc_library(
+    name = "s6a_proxy_cpp_grpc",
+    protos = [":s6a_proxy_proto"],
+    deps = [":s6a_proxy_cpp_proto"],
```

Comment:
For Python this dependency is not necessary - the grpc rule creates proto and grpc libraries. Is this needed for cpp?